apis/v1alpha1/key.go

```diff
@@ -196,6 +163,9 @@ type KeyStatus struct {
 	// The date and time when the CMK was created.
 	// +kubebuilder:validation:Optional
 	CreationDate *metav1.Time `json:"creationDate,omitempty"`
+	// Describes the type of key material in the CMK.
+	// +kubebuilder:validation:Optional
+	CustomerMasterKeySpec *string `json:"customerMasterKeySpec,omitempty"`
```

Comment:
I'm not sure. I guess since you can't set it in the Spec, it shouldn't appear in the Status either? I added it to the ignore since it is deprecated in the API.

Reply:
>I guess since you can't set it in the Spec, it shouldn't appear in the Status either?

Yes, I agree. The field should be ignored completely IMO.